packages/user/src/types/invitation.ts

```diff
@@ -33,6 +36,20 @@ export interface InvitationAppOption {
   supportedRoles: InvitationRoleOption[];
 }
 
+export interface AdditionalInvitationFields {
+  renderFields: RenderAdditionalInvitationFields;
+  additionalInvitationSchema: AdditionalInvitationSchema;
+  additionalDefaultValues: AdditionalDefaultValues;
```

Comment:
I think there's no need for explicitly naming additional for these attributes. Since they are under AdditionalInvitationFields, we can simply say `schema` and `default values`.
```suggestion
  schema: AdditionalInvitationSchema;
  defaultValues: AdditionalDefaultValues;
```